requirements.txt

```diff
@@ -20,7 +20,7 @@ twine==4.0.2
 
 # Tests & Linting
 black==23.3.0
-coverage==7.2.2
+coverage[toml]==7.2.2
```

Comment:
Why did you add this?

Reply:
exactly. from what I have seen, coverage requires `toml` extra to work with `pyproject.toml` configs.